qbft/controller.go

```diff
@@ -105,16 +105,16 @@ func (c *Controller) UponExistingInstanceMsg(msg *ProcessingMessage) (*types.Sig
 
 	prevDecided, _ := inst.IsDecided()
 
+	// if previously decided, we don't process more messages
+	if prevDecided {
```

Comment:
does this include only consensus messages? we want to keep processing post-consensus message.s.

Reply:
Yup, includes only consensus messages because this is already at the `qbft.Controller` level, so no partial sig msg should reach here